argoverse/config/argoverse1.1.yaml

```diff
@@ -0,0 +1,54 @@
+SensorDatasetConfig:
```

Comment:
Do we need this to be nested?

Reply:
I think the hierarchy makes it a bit more readable -- a class with 4 fields instead of 11